devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/gradle/codestart.yml

```diff
@@ -19,9 +19,10 @@ language:
         cmd:
           dev: gradle quarkusDev
           package: gradle quarkusBuild
-          package-uberjar:  gradle quarkusBuild --uber-jar
+          package-uberjar: gradle quarkusBuild --uber-jar
           package-native: gradle build -Dquarkus.package.type=native
           package-native-container: gradle build -Dquarkus.package.type=native -Dquarkus.native.container-build=true
+          build-ci: gradle build
```

Comment:
this shuold be `./gradlew build` to ensure it uses the right gradle

Reply:
same as below - need to check we get the right/new enough version of gradle in the build

Reply:
I will fix that in another iteration, I need this to make master work with code.quarkus

Reply:
https://github.com/quarkusio/quarkus/issues/12438